src/coreclr/jit/valuenumfuncs.h

```diff
@@ -136,6 +136,7 @@ ValueNumFuncDef(JitNewArr, 3, false, true, false)
 ValueNumFuncDef(JitNewMdArr, 4, false, true, false)
 ValueNumFuncDef(JitReadyToRunNew, 2, false, true, false)
 ValueNumFuncDef(JitReadyToRunNewArr, 3, false, true, false)
+ValueNumFuncDef(BoxSlow, 3, false, false, false)
```

Comment:
This does not need its own value number. It can just use the `Box` value number. (It is similar to how different perf variants of `JIT_New` share same value number.